GoogleSignIn/Sources/GIDEMMErrorHandler.h

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <TargetConditionals.h>
+
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
```

Comment:
Instead of conditionally including everything in the header, can we conditionally import this and other EMM headers where they are used?

Bonus point if we can conditionally include the EMM files at the dependency manager level (podspec and Package.swift) too.

Reply:
These headers are only imported when they are needed in the next PR. I see your point here. I will revert this PR.

Reply:
I already did that in the next PR. I guess I can revert this PR.

Reply:
Reopen this PR to only modify configuration.